### ISSUE TITLE
The accuracy that I have done for RoBERTa-large fine-tuning on RACE has a big gap with the paper proposed

### ISSUE THREAD

**Issue by 14H034160212** [pytorch/fairseq#1946]
## 🐛 Bug

Hi, 

I have tried to use 2 Tesla V100 GPU with 32 GB memory to fine-tuning the RoBERTa-large model with RACE dataset. Here is the part of the result. Now I have finished the running with 5 epochs, but the loss and accuracy look like nearly no change. Does anyone have met the problem before? I just use the exact same parameters from that website for the fine-tuning on RACE dataset(https://github.com/pytorch/fairseq/blob/master/examples/roberta/README.race.md). Will be great to have any suggestions from you. 

Many thanks.

Here is the command and parameters that I used.
```
MAX_EPOCH=5           # Number of training epochs.
LR=1e-05              # Peak LR for fixed LR scheduler.
NUM_CLASSES=4
MAX_SENTENCES=1       # Batch size per GPU.
UPDATE_FREQ=8         # Accumulate gradients to simulate training on 8 GPUs.
DATA_DIR=/path/to/race-output-dir
ROBERTA_PATH=/path/to/roberta/model.pt

CUDA_VISIBLE_DEVICES=0,1 fairseq-train $DATA_DIR --ddp-backend=no_c10d \
    --restore-file $ROBERTA_PATH \
    --reset-optimizer --reset-dataloader --reset-meters \
    --best-checkpoint-metric accuracy --maximize-best-checkpoint-metric \
    --task sentence_ranking \
    --num-classes $NUM_CLASSES \
    --init-token 0 --separator-token 2 \
    --max-option-length 128 \
    --max-positions 512 \
    --truncate-sequence \
    --arch roberta_large \
    --dropout 0.1 --attention-dropout 0.1 --weight-decay 0.01 \
    --criterion sentence_ranking \
    --optimizer adam --adam-betas '(0.9, 0.98)' --adam-eps 1e-06 \
    --clip-norm 0.0 \
    --lr-scheduler fixed --lr $LR \
    --fp16 --fp16-init-scale 4 --threshold-loss-scale 1 --fp16-scale-window 128 \
    --max-sentences $MAX_SENTENCES \
    --required-batch-size-multiple 1 \
    --update-freq $UPDATE_FREQ \
    --max-epoch $MAX_EPOCH
```

Here is the final training result after 5 epoch. The best accuracy is only 0.261661.

![roberta_large_race_result_epoch_5_final](https://user-images.githubusercontent.com/23516191/78081272-bc77ea80-73e2-11ea-8e79-d555967bf8ae.png)

Here is the evaluation result and the accuracy is only 0.245.
![roberta_large_evaluation_result](https://user-images.githubusercontent.com/23516191/78084957-1fba4a80-73ec-11ea-9018-226355769621.png)

**Comment by 14H034160212** [comment]
I seemly find the solution from that link. https://github.com/pytorch/fairseq/issues/1114
It worth to have a try.
